### PR TITLE
Edit/update docs general

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

No

### Please describe the goal of this pull request and the changes that were made.

Apologies this started off as a collection of small things but then I ended up digging into Style Customization. Would have been better served as separate PRs.

- update install/uninstall guidelines per v1
- update PDF styling per feedback from Zsofia 
- Update advice for IIIF 
- Add new recipe per advice given to Zsofia

### CODE REVIEW

File: content/docs-v1/site-deploy.md [hunk 1]
```diff
@@ -9,6 +9,19 @@ aliases:
 
 A Quire site is designed to be hosted on virtually any web server, either one your institution already runs, or a new server from the hosting service of your choice. You do not need any special back-end setup. If you plan to include PDF and e-book files as part of you published project, it's important that you start by following the directions in the [*Output Your Project*](/docs-v1/multiformat-output/) section of the documentation before proceeding.
 
+Before deploying your project, you can run a test to make sure all the `_site` files are building correctly and that your styles are properly loading:
+
+1. *Temporarily* set the URL in `publication.yaml` to [http://localhost:8000/](http://localhost:8000/)
+2. Delete `public`, `_site`, and `epub`
```
Comment (geealbers) [inline content/docs-v1/site-deploy.md:15]: underscore before epub: `_epub`

File: content/resources/recipes.md [hunk 2]
```diff
@@ -6,4 +6,5 @@ abstract: "Step-by-step guides on creating popular customizations to your projec
 ---
 
 - [Tag Select Pages in the Sidebar with a Custom Icon](/resources/recipes-menu-icon/)
-- [Hide Title and Subtitle from Cover Page with Custom CSS](/resources/recipes-cover/)
\ No newline at end of file
+- [Hide Title and Subtitle from Cover Page with Custom CSS](/resources/recipes-cover/)
+- [Hide Table of Contents Grid in PDF with Custom Class](/resources/hide-grid)
```
Comment (geealbers) [inline content/resources/recipes.md:10]: URL may need slash at the end: `/resources/hide-grid/`

**Review by geealbers — CHANGES_REQUESTED** [review]
Just caught a couple very small things

**Comment by Erin-Cecele** [comment]
Done!

**Review by geealbers — APPROVED** [review]

**Pull request merged**